eras/shelley/impl/CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
 * Add `shelleyGenesisKeyHashCount`
 * Change `PPUP` signal to `StrictMaybe` from `Maybe`
 * Deprecate `txup` function.
-* Prevent instances for `ShelleyEraTxBody` starting with protocol version 9.
+* Prevent instances for `SahelleyEraTxBody` starting with protocol version 9.
```

Comment:
Introduced a spurious `a` here in `ShelleyEraTxBody` :)

Reply:
Thank you. Fixed